shotover-proxy/src/frame/cassandra.rs

```diff
@@ -476,38 +505,292 @@ impl CassandraOperation {
 }
 
 #[derive(PartialEq, Debug, Clone)]
-pub enum CQL {
-    // Box is used because Statement is very large
-    Parsed(Box<Statement>),
-    FailedToParse(String),
+pub struct CQLStatement {
```

Comment:
Now that we have removed has_error the only purpose of CQLStatement is to provide the get_query_type method, that could very easily be made a function instead of a method and then we can remove CQLStatement entirely.

Reply:
No point in keeping around an empty struct, the functions can just exist in the module without the `impl CQLStatement`